pkgs/shells/zsh/agdsn-zsh-config/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, zsh
+, coreutils
+, inetutils
+, procps
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  pname = "agdsn-zsh-config";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "agdsn";
+    repo = "agdsn-zsh-config";
+    rev = "v${version}";
+    sha256 = "sha256-/l2fE4ZsZ6f89fYG9sTEV1mrXZ3MLXx6K3CTUQHiAsc=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -D -m644 zshrc-base-hw.zsh $out/etc/zsh/zshrc
+    install -D -m644 zshrc-home.zsh $out/etc/skel/.zshrc
+    install -D -m644 zshrc-home.zsh $out/etc/zsh/newuser.zshrc.recommended
```

Comment:
Why the target paths diverge from https://github.com/agdsn/agdsn-zsh-config/blob/master/debian/agdsn-zsh-config.install

Reply:
I spoke with the upstream maintainer personally, he told me that the `-agdsn` suffix is a hack because on Debian these files would conflict with the ones zsh ships itself, and that I should use the default paths if possible.